src/compose/commit.ts

```diff
@@ -0,0 +1,32 @@
+import * as db from '../db';
+
+const cache: { [appId: number]: string } = {};
+
+export async function getCommitForApp(
+	appId: number,
```

Comment:
We potentially pass `undefined` to this method, would this impact the `cache[appId]` part?

Reply:
Good catch.